Exercise1_cheat2.py

```diff
@@ -0,0 +1,29 @@
+def withdraw_money(account_balance, amount):
+  """Withdraws money from a bank account.
+
+  Args:
+    account_balance: The current account balance.
+    amount: The amount of money to withdraw.
+
+  Returns:
+    The updated account balance.
+  """
+
+  if amount > account_balance:
+    print("No money")
+    return account_balance
+  else:
+    print("Your money is processed and thanks for visiting the bank")
+    return account_balance - amount
+
+# Get the account balance
+account_balance = 4000
+
+# Ask the user for the amount to withdraw
+amount = float(input("How much money do you want to withdraw? "))
```

Comment:
this is correct, but it can lead to issues in future. the problem is that the user can input "yes" and float("yes") wont make any sense to python.